pkg/machine/qemu/machine.go

```diff
@@ -1127,11 +1141,13 @@ func (v *MachineVM) startHostNetworking() (string, machine.APIForwardingState, e
 		logrus.Debug(cmd)
 	}
 
+	cargs := cmd.ToCmdline()
+	logrus.Debugf("gvproxy cmd: %v", append([]string{binary}, cargs...))
 	c := cmd.Cmd(binary)
 	if err := c.Start(); err != nil {
-		return "", 0, fmt.Errorf("unable to execute: %q: %w", cmd.ToCmdline(), err)
+		return "", 0, nil, fmt.Errorf("unable to execute: %q: %w", cmd.ToCmdline(), err)
 	}
-	return forwardSock, state, nil
+	return forwardSock, state, c.Process, nil
```

Comment:
What was the decision behind returning the `os.Process`? it currently isn't being used by the caller

Reply:
I have implementation based of UNIX socket VLAN based which checks the pid of this process to understand if we have the process alive https://github.com/arixmkii/podman/commit/9e3e20fd53505cfd3e7ac3bce2253ab6871c448a#diff-4a3677ed5171641eba40e911f19bbf5970550a9e1ce75a30959e6c8032b2e39cR542